src/entities/media-sources.ts

```diff
@@ -23,6 +24,7 @@ export default class MediaSources {
    */
   public hls: Array<MediaSource>;
   public image: Array<ImageSource>;
+  public doc: Array<DocSource>;
```

Comment:
i think the full name (document) would be clearer
unless its coming as doc from the backend ?